ingenialink/pdo.py

```diff
@@ -202,28 +209,48 @@ class PDOMap:
     def __init__(self) -> None:
         self.__items: list[PDOMapItem] = []
         self.__map_register_address: Optional[int] = None
+        self.__slave: Optional[PDOServo] = None
+
+    @property
+    def slave(self) -> Optional["PDOServo"]:
+        """Servo state referece - allows to get the status of the network."""
```

Comment:
This does not get the state, and it's not relevant what this property is used for.

I would state: "Servo to which this PDO is mapped."